systers_portal/templates/404.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block title %} - Page Not Found{% endblock %}
+
+{% block content %}
+  
+  <div class="jumbotron jumbotron-custom">
+    <h1>404 Error - Page Not Found</h1>
```

Comment:
The word "Error" sounds a bit scary to the user. Though it is in reality an HTTP error code, the user can have bad associations with it.

My suggestion: what if we remove the "Error" word at all? Just leave "404". What do you think about that?
Also maybe put "404" and "Page Not Found" on different lines? For example create two headings inside the jumbotron.

**How to update this pull request**
If you decide to apply any of the changes, here is the simplest scenario to update this pull request:
1. Make the change on this branch `404_page`
2. `git add <the_file>`
3. `git commit --amend`
4. `git push -f origin 404_page`

This pull request will update automatically according to the pushed branche, which will override the old commit with the new one (the SHA1 changes when doing amend).